src/status_im/contexts/wallet/send/flow_config.cljs

```diff
@@ -1,13 +1,30 @@
 (ns status-im.contexts.wallet.send.flow-config)
 
+(defn- collectible-selected?
+  [db]
+  (let [collectible-stored (-> db :wallet :ui :send :collectible)
+        tx-type            (-> db :wallet :ui :send :tx-type)]
+    (and (some? collectible-stored)
+         (= tx-type :collectible))))
+
+(defn- token-selected?
+  [db]
+  (-> db :wallet :ui :send :token some?))
+
 (def steps
   [{:screen-id  :screen/wallet.select-from
     :skip-step? (fn [db] (some? (get-in db [:wallet :current-viewing-account-address])))}
    {:screen-id  :screen/wallet.select-address
     :skip-step? (fn [db] (some? (get-in db [:wallet :ui :send :recipient])))}
    {:screen-id  :screen/wallet.select-asset
-    :skip-step? (fn [db] (some? (get-in db [:wallet :ui :send :token])))}
+    :skip-step? (fn [db] (or (token-selected? db) (collectible-selected? db)))}
    {:screen-id  :screen/wallet.send-input-amount
-    :skip-step? (fn [db] (some? (get-in db [:wallet :ui :send :amount])))}
+    :skip-step? (fn [db]
+                  (or (not (token-selected? db))
+                      (some? (get-in db [:wallet :ui :send :amount]))))}
+   {:screen-id  :screen/wallet.select-collectible-amount
+    :skip-step? (fn [db]
+                  (or (not (collectible-selected? db))
+                      (some? (get-in db [:wallet :ui :send :amount]))))}
```

Comment:
`amount` should be greater than 1, so probably we should skip if amount == 1 🤔

Reply:
we have multiple collectibles now :)

Reply:
this `amount` is not the owned collectible units, this `amont` key represents the desired number of collectibles to send and we are skipping this step if it already exists in the db.
i.e.
During this step we set the value in `[:wallet :ui :send :amount]`, if it has been already set, we skip it.

Reply:
yes, but the user can decide how many at a time they want to send. 
see @vkjr's recent pr:
https://github.com/status-im/status-mobile/pull/19386

Reply:
i,e https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=10771-86609&mode=design&t=XuAtw4Ji54tLkXJS-4

although, maybe we're talking about the same thing? 
🤔 
in any case this should only be for collectibles that are owned, so if this is a stretch in functionality, let's do a follow up instead.

Reply:
> this `amount` is not the owned collectible units, this `amont` key represents the desired number of collectibles to send and we are skipping this step if it already exists in the db. i.e. During this step we set the value in `[:wallet :ui :send :amount]`, if it has been already set, we skip it.

I see, thanks for the explanation @ulisesmac.

> yes, but the user can decide how many at a time they want to send. see @vkjr's recent pr: #19386

What I meant is that if user only owns 1 of the collectibles, it doesn't make sense to show `:screen/wallet.select-collectible-amount`. But I see the case is already covered by this part
```
     {:db (cond-> collectible-tx
            one-collectible? (assoc-in [:wallet :ui :send :amount] 1))
```
in `:wallet/set-collectible-to-send` event, and the condition in skip-step?.

Looks good to me 👍

Reply:
@J-Son89 
Yeah, that flow is being preserved, and it's only shown when the user has more than one collectible.

ok! @briansztamfater  :)